scripts/load_location_data.rb

```diff
@@ -0,0 +1,229 @@
+  {:name => 'archive half legal', :depth => '15.5', :height => '10.25', :width => '2.5', :dimension_units => 'inches', :extent_dimension => 'width'},
+  {:name => 'Flat Box', :width => '15', :height => '3', :depth => '18.5', :dimension_units => 'inches', :extent_dimension => 'width'},
+  {:name => 'Oversize Folder - half case', :depth => '35.75', :height => '0.15', :width => '23.75', :dimension_units => 'inches', :extent_dimension => 'width'},
+  {:name => 'Oversize Folder - full case', :depth => '35.75', :height => '0.15', :width => '48', :dimension_units => 'inches', :extent_dimension => 'width'},
+  {:name => 'CD ', :width => '0.4', :height => '4.92', :depth => '5.59', :dimension_units => 'inches', :extent_dimension => 'width'},
+  {:name => 'DVD ', :width => '0.55', :height => '7.6', :depth => '5.4', :dimension_units => 'inches', :extent_dimension => 'width'}
+]
+
+
+# Some packings that will mostly fill the different location profile types.
+#
+# Of the form:
+#
+#  { 'location profile name' => [packing1, packing2, packing3] }
+#
+# where each 'packing' is a list of pairs like:
+#
+#  ['container profile name', number_of_boxes]
+#
+#
+SAMPLE_LOCATION_PROFILE_PACKINGS = {
+  "record carton shelf (Remarque)" =>
+  [[["archive half legal", 6], ["Paige 15", 1], ["archive legal", 2]],
+   [["archive half legal", 15]]],
+
+  "oversize - short - TAM" =>
+  [[["Flat Box", 4]]],
+
+  "oversize - short - TAM 2" =>
+  [[["Flat Box", 2], ["CD ", 23]]],
+
+  "2 deep - 3 wide - 1 high" =>
+  [[["Paige 15", 6]],
+   [["archive legal", 4], ["archive letter", 8], ["archive half legal", 6]]],
+
+  "2 deep - 3 wide - 2 high" =>
+  [[["Flat Box", 8], ["archive half legal", 12]]],
+
+  "oversize - short" =>
+  [[["Oversize Folder - half case", 29]]],
+
+  "cd shelf 1" =>
+  [[["CD ", 90]]],
+
+  "cd shelf 2" =>
+  [[["DVD ", 42]]],
+
+  "Fales Media Storage 1" =>
+  [[["archive letter", 6]]],
+
+  "Fales Media Storage 2" =>
+  [[["archive letter", 6]]],
+
+  "map case 2" =>
+  [[["Oversize Folder - full case", 9]]],
+
+  "map case 3" =>
+  [[["Oversize Folder - half case", 12]]],
+
+  "CSQ standard compact shelf" =>
+  [[["Paige 15", 1], ["archive legal", 2]]],
+
+  "CSQ extra high compact shelf with set back" =>
+  [[["archive half legal", 11]]],
+
+  "CSQ extra high compact shelf with no set back" =>
+  [[["Paige 15", 2]]],
+
+  "UA Shelving (varies)" =>
+  [[["archive letter", 7]]],
+
+  "Fales 10W - 2 wide" =>
+  [[["Paige 15", 1], ["archive letter", 2]]]}
+
+# Place names
+PLACES = java.util.TimeZone.getAvailableIDs.map {|z| z.to_s.split('/')[1]}.compact.reject {|p| p =~ /[A-Z]{3}/}.map {|p| p.sub('_', ' ')}.uniq
+
+# Suffixes
+LOCATION_SUFFIXES = ['Square', 'Gardens', 'Library', 'Museum', 'Warehouse', 'Treehouse', 'Bunker']
+
+
+require 'jsonmodel'
+require 'securerandom'
+
+JSONModel::init(:client_mode => true, :url => 'http://localhost:4567')
```

Comment:
Would it be useful to make the URL here configurable?  Do people run on separate machines or use different port numbers?